app/Libraries/OsuAuthorize.php

```diff
@@ -1788,6 +1788,19 @@ public function checkIsSpecialScope(?User $user): string
         return 'unauthorized';
     }
 
+    public function checkLegacyIrcKeyStore(?User $user): string
+    {
+        $this->ensureLoggedIn($user);
+        $this->ensureCleanRecord($user);
+
+        // isBot checks user primary group
+        if (!$user->isGroup(app('groups')->byIdentifier('bot')) && $user->playCount() < 100) {
```

Comment:
I'm not sure if we want to change this from being primary group only? @peppy

Reply:
after looking at the requirement apparently you can't even be in bot group before running a bot first so this check probably doesn't matter whether or not it's primary group (the current one in old site doesn't require primary group).

Also having main account being in bot group isn't even a thing anymore anyway? (at least no new one)

Reply:
Skipping the playcount check on the old site seems to only apply to users where the primary group is the bot group

Reply:
the query is on `phpbb_user_group` 🤔